### PR TITLE
Fix listdebtors showing persons with all debts marked

### PR DESCRIPTION
**Pull request by lyuting47**
Listdebtors without amount specified will no longer list persons with all their debts marked

### CODE REVIEW

**Review by waynezsy — DISMISSED** [review]
LGTM

File: src/main/java/paymelah/logic/commands/ListDebtorsCommand.java [hunk 1]
```diff
@@ -26,6 +27,7 @@ public class ListDebtorsCommand extends Command {
     public static final String MESSAGE_SUCCESS_SPECIFIC = "Listed all persons with debts amounting to more than $%1$s.";
     public static final String MESSAGE_SUCCESS_GENERIC = "Listed all persons with debts.";
     public static final String MESSAGE_NO_DEBTORS = "There are no persons with debts";
+    public static final Money ZERO_MONEY = new Money("0");
```
Comment (tsammeow) [inline src/main/java/paymelah/logic/commands/ListDebtorsCommand.java:30]: Should this be a constant of Money instead of ListDebtorsCommand?

**Review by tsammeow — CHANGES_REQUESTED** [review]

File: src/main/java/paymelah/logic/commands/ListDebtorsCommand.java (hunk 1 above)
Comment (lyuting47) [reply src/main/java/paymelah/logic/commands/ListDebtorsCommand.java:30]: ok will change

**Review by tsammeow — APPROVED** [review]
LGTM!

**Pull request merged**